### PR TITLE
feat: M2-1013 - Activity Settings - Left Bar

### PR DESCRIPTION
**Pull request by muhurov**

### CODE REVIEW

File: src/modules/Builder/features/BuilderAppletSettings/BuilderAppletSettings.tsx [hunk 1]
```diff
@@ -1,13 +1,27 @@
...
+import { useBreadcrumbs } from 'shared/hooks';
 
-import { getSettings } from './BuilderAppletSettings.const';
+import { LeftBar } from './LeftBar';
+import { StyledWrapper } from './BuilderAppletSettings.styles';
 
 export const BuilderAppletSettings = () => {
-  const location = useLocation();
-  const isEditAppletPage = location.pathname.includes(page.newApplet);
+  const [activeSetting, setActiveSetting] = useState('scoresAndReports');
```
Comment (alex-i-e) [inline src/modules/Builder/features/BuilderAppletSettings/BuilderAppletSettings.tsx:11]: let's use enum (with subscalesConfiguration)

File: src/modules/Builder/features/BuilderAppletSettings/LeftBar/Item/Item.styles.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  theme,
+  variables,
+  StyledFlexColumn,
+  StyledFlexTopCenter,
+  StyledTitleBoldMedium,
+  StyledTitleMedium,
+} from 'shared/styles';
+import { shouldForwardProp } from 'shared/utils';
+
+const commonEllipsisStyles = `
```
Comment (alex-i-e) [inline src/modules/Builder/features/BuilderAppletSettings/LeftBar/Item/Item.styles.ts:13]: please add these common styles into `src/shared/styles/stylesConsts.ts` , they are also used in StyledEllipsisText

**Review by alex-i-e — APPROVED** [review]

File: src/svgSprite.ts [hunk 3]
```diff
@@ -961,5 +961,14 @@ export const svgSprite = (): string => `
...
+  </symbol>
+
+  <symbol viewBox="0 0 20 20" id="svg-grid-outlined">
+    <path d="M19 0H1C0.734784 0 0.48043 0.105357 0.292893 0.292893C0.105357 0.48043 0 0.734784 0 1V19C0 19.2652 0.105357 19.5196 0.292893 19.7071C0.48043 19.8946 0.734784 20 1 20H19C19.2652 20 19.5196 19.8946 19.7071 19.7071C19.8946 19.5196 20 19.2652 20 19V1C20 0.734784 19.8946 0.48043 19.7071 0.292893C19.5196 0.105357 19.2652 0 19 0ZM6 18H2V14H6V18ZM6 12H2V8H6V12ZM6 6H2V2H6V6ZM12 18H8V14H12V18ZM12 12H8V8H12V12ZM12 6H8V2H12V6ZM18 18H14V14H18V18ZM18 12H14V8H18V12ZM18 6H14V2H18V6Z" fill="#42474E"/>
```
Comment (IhorNesterenko99) [inline src/svgSprite.ts:970]: pls use without fill and add fill with styles

File: src/svgSprite.ts [hunk 4]
```diff
@@ -961,5 +961,14 @@ export const svgSprite = (): string => `
...
+  </symbol>
+
+  <symbol viewBox="0 0 20 20" id="svg-grid-outlined">
```
Comment (IhorNesterenko99) [inline src/svgSprite.ts:969]: why viewBox is not as usual?

File: src/svgSprite.ts (hunk 3 above)
Comment (muhurov) [reply src/svgSprite.ts]: fixed

File: src/svgSprite.ts [hunk 5]
```diff
@@ -961,5 +961,14 @@ export const svgSprite = (): string => `
     <path d="M7.66326 3.875L9.85294 5.25C10.2407 5.5 10.3547 6.025 10.1495 6.425C10.081 6.55 9.98979 6.65 9.87575 6.725L7.66326 8.125C7.54922 8.2 7.41236 8.25 7.27551 8.25C6.84214 8.25 6.5 7.875 6.5 7.4V4.6C6.5 4.125 6.84214 3.75 7.27551 3.75C7.41236 3.75 7.54922 3.8 7.66326 3.875Z"/>
     <path fill-rule="evenodd" clip-rule="evenodd" d="M2.75 0C1.50736 0 0.5 1.00736 0.5 2.25V9.75C0.5 10.9926 1.50736 12 2.75 12H13.25C14.4926 12 15.5 10.9926 15.5 9.75V2.25C15.5 1.00736 14.4926 0 13.25 0H2.75ZM13.25 1.5H2.75C2.33579 1.5 2 1.83579 2 2.25V9.75C2 10.1642 2.33579 10.5 2.75 10.5H13.25C13.6642 10.5 14 10.1642 14 9.75V2.25C14 1.83579 13.6642 1.5 13.25 1.5Z"/>
   </symbol>
+
+  <symbol viewBox="0 0 16 20" id="svg-scores-and-reports">
```
[the remark this answers is not in the document]
Comment (muhurov) [reply src/svgSprite.ts]: fixed

File: src/svgSprite.ts (hunk 4 above)
Comment (muhurov) [reply src/svgSprite.ts]: fixed

**Pull request merged**